src/backend/marsha/core/management/commands/transcript_video.py

```diff
@@ -0,0 +1,54 @@
+"""Management command to transcript a video."""
+
+from django.core.management import BaseCommand
+
+from marsha.core import defaults
+from marsha.core.models import TimedTextTrack, Video
+from marsha.core.utils.transcript_utils import transcript
+
+
+class Command(BaseCommand):
+    """Transcript a video."""
+
+    help = "Transcript a video"
+
+    def add_arguments(self, parser):
+        parser.add_argument("--video-id", type=str)
+
+    def handle(self, *args, **options):
+        """Selects a video to transcript and starts the transcription job."""
+        video_id = options["video_id"]
+        if video_id:
+            try:
+                video = Video.objects.get(id=video_id)
+            except Video.DoesNotExist:
```

Comment:
I don't know if we should catch it or not. IMO I would like to know that the video_id is not valid.

Reply:
A message is displayed, and the script is stopped.

            except Video.DoesNotExist:
                self.stdout.write(f"No video found with id {video_id}")
                return

Reply:
If the video_id is invalid (not a uuid) another message will be displayed, and the script will be also stopped.

Reply:
Yes but this command will be probably trigger by another one, especially if the `video_id` optional argument will be used. And we will have no idea that is has fail

Reply:
I don't like nested command calls, especially in this case where the main logic only calls 

    from marsha.core.utils.transcript_utils import transcript
    transcript(video)

We have to talk about whole catalog transcription.